scripts/pin-external-packages.sh

```diff
@@ -1,8 +1,11 @@
 #!/bin/sh
 
-# update and pin packages, used by CI
+# Extlib gets automatically installed, but we want our pin, so we should
+# uninstall here
+opam uninstall extlib
 
-PACKAGES="ocaml-sodium rpc_parallel ocaml-extlib ocaml-extlib async_kernel coda_base58 graphql_ppx"
```

Comment:
after our discussion @mrmr1993 I'm still unsure if `coda_base58` needs to be re-introduced. There is no `coda_base58.opam` and no `dune` file there, just an `opam` with `name: "base58"`.

So my guess is that, to be used it needs to be installed by opam (otherwise dune can't find it, due to the lack of `dune` file AND `base58.opam` file.

So that it can be installed by opam, it needs to be pinned.

Reply:
On the other hand there's a `graphql_ppx.opam` and `dune` file with `(name graphql_ppx)` so there should be no problem there.